src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx

```diff
@@ -0,0 +1,518 @@
+between service meshes and which one might be best-suited for specific workloads. Individual service mesh projects shy from publishing test 
+results of other, competing service mesh projects. The need for an independent, unbiased, credible, standard measurement is one of the catalysts 
+for the creation of Service Mesh Performance (SMP).
+</p>
+
+<p>
+Amidst performance concerns and the need to measure and manage performance arose the Service Mesh Performance (SMP) standard. Service Mesh 
+Performance as a specification and disseminating insights and research results. Your authors are working toward the definition of MeshMark, 
+a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments.
+</p>
+
+<p>
+Many performance benchmarks are limited to single instance load generation (single pod load generator). This limits the amount of traffic 
+that can be generated to the output of the single machine that the benchmark tool runs on in or out of a cluster. Overcoming this 
+limitation would allow for more flexible and robust testing. Distributed load testing in parallel poses a challenge when merging 
+results without losing the precision we need to gain insight into the high tail percentiles. Distributed load testing offers insight 
+into system behaviours that arguably more accurately represent real-world behaviours of services under load as that load comes from 
+any number of sources.
+</p>
+
+<p>
+The specification itself provides a standard format for describing and capturing:
+</p>
+
+<ul>
+  <li>performance test configuration</li>
+  <li>Pick a benchmarking tool that can measure Layer 4 or Layer 7 performance.</li>
+  <li>service mesh configuration</li>
+  <li>environment configuration</li>
+  <li>workload configuration</li>
+  <li>performance test results</li>
+  <li>Distributed performance modeling</li>
+  <li>KPIs for service mesh performance</li>
+  <li>Test tool requirements</li>
+</ul>
+
+<p>
+Value from a service mesh is best derived when it's tuned to scale as per the deployment requirements. Given the 
+complexity of deploying, testing and measuring performance aspects across multiple dimensions, the specification 
+aims to provide a simple starting point for anyone looking to understand and derive service mesh performance. The 
+service mesh performance standard aims to articulate these complexities in a methodical and automated manner in 
+order for anyone to plan the performance scenarios of their deployment and execute relevant tests.
+</p>
+
+<p>
+Figure 2 provides insight to the fact that the specification defines a common collection of statistical analysis 
+to be calculated for every performance test.
+</p>
+
+<pre>
+  <code>
+    message PerformanceTestResult {
+    message Latency {
+    double min = 1;
+    double average = 2;
+    double p50 = 3;
+    double p90 = 4;
+    double p99 = 5;
+    double max = 6;
+  }

```

Comment:
```suggestion
  }
}
```

Reply:
@Mr-DG-Wick I think the build is failing due to the missing `}` here. Can you add that in to verify?